### ISSUE TITLE
Trashbin is not working on android after RC upgrade

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/server#10905]
First dav error is quite strange, as this does not happen with another user.

```
<?xml version="1.0" encoding="utf-8"?>
<d:error xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns">
  <s:exception>Sabre\DAV\Exception</s:exception>
  <s:message>The REQUEST_URI (/remote.php/dav/trashbin/tobi/trash//) did not end with the contents of PATH_INFO (/dav/trashbin/tobi/trash/). This server might be misconfigured.</s:message>
</d:error>
```

After changing url to match "/dav/trashbin/tobi/trash/" instead of "/dav/trashbin/tobi/trash//", I get this one:

```
<?xml version="1.0" encoding="utf-8"?>
<d:error xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns">
  <s:exception>Sabre\DAV\Exception\ServiceUnavailable</s:exception>
  <s:message>TypeError: Return value of OCA\Files_Trashbin\Sabre\TrashFile::getOriginalLocation() must be of the type string, null returned</s:message>
</d:error>

```

Difference between those two users are, that one (tobi) has deleted folders in it, whereas the other (test) has only files.

@rullzer as discussed, please have a look.

**Comment by tobiasKaminsky** [comment]
Why do you think it is fixed? This is still an error for me on Android 3.3.0 and reproducible with one of my user (tobi), whereas the other one (test) is working fine.
@rullzer can you help me debugging this?

**Comment by rullzer** [comment]
Lets see if this is fixed with 14.0.2 as we fixed some init there

**Comment by tobiasKaminsky** [comment]
This is still happening with 14.0.2:

user test: /remote.php/dav/trashbin/test/trash//
user tobi: /remote.php/dav/trashbin/tobi/trash//

But user "test" is working.
